src/promptflow/promptflow/_cli/_pf/_init_entry_generators.py

```diff
@@ -232,11 +232,13 @@ def flow_inputs(self):
         executable = ExecutableFlow.from_yaml(
             flow_file=Path(self.flow_dag_path.name), working_dir=self.flow_dag_path.parent
         )
-        return {flow_input: (value.default, value.type.value) for flow_input, value in executable.inputs.items()}
+        return {flow_input: (value.default, value.type.value) for flow_input, value in executable.inputs.items()
+                if flow_input != "chat_history"}
 
     @property
     def flow_inputs_params(self):
-        flow_inputs_params = ["=".join([flow_input, flow_input]) for flow_input, _ in self.flow_inputs.items()]
+        flow_inputs_params = ["=".join([flow_input, flow_input]) for flow_input, _ in self.flow_inputs.items()
+                              if flow_input != "chat_history"]
```

Comment:
The name of chat history may not be chat_history. Need change to
```
["=".join([flow_input, flow_input]) for flow_input, value in self.flow_inputs.items() if not value.is_chat_history]
```

Reply:
Thanks, @brynn-code, according to the discussion yesterday, may I ask we ignore `chat_history` input or all flow inputs with `is_chat_history` is True?

Reply:
Yeah we may support chat history in the next pr, no rush to this one, as our test page (the html one) has a hard code key for now. I remember we have a task, we may use that to track this.

Reply:
keep `flow_input != "chat_history"` behavior for now